InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/shooter/ShooterSubsystem.java

```diff
@@ -127,4 +133,29 @@ public void rotateToDeg(double targetPoint) {
         targetChange = 0;
     }
 
-}
\ No newline at end of file
+    public void rotateByDeg(double degrees) {
+        rotateToDeg(getTargetTurretDegGivenOffset(degrees));
+    }
+    
+    public double getTurretDeg() {
+        double encoderDeg = MathUtils.unitConverter(azimuthMotor.getSelectedSensorPosition(), config.shooter.azimuth.ticksPerRevolution, 360.0);
+        double turretDeg = encoderDeg * config.shooter.gearRatio;
+        return turretDeg;
+    }
+
+    /*
+        Returns target degrees of turret given an offset
+        Assuming both sides agree that "left" is a negative offset.
+    */
+    public double getTargetTurretDegGivenOffset(double offset) {
+        return getTurretDeg() + offset;

```

Comment:
if we assume a negative offset means to move left (or counterclockwise), then the original math was right because the turret spinned CCW given a positive voltage. we may want to put a boolean value in the shooterconstants representing the direction it spins in in case they change the hardware, but that part can wait until after I merge your changes